new-site/src/components/Hero.scss

```diff
@@ -0,0 +1,149 @@
+@import "./utils.scss";
+
+.hero-container {
+  --koros-height: 145px;
+  --hero-height: 540px;
+  --hero-small-left: calc((100vw - 100%) / 2);
+  --hero-large-content-left: calc((100vw + var(--spacing-s) - var(--container-width-xl)) / 2);
+
+  background-color: var(--color-coat-of-arms);
+  box-sizing: border-box;
+  position: relative;
+  left: calc(var(--hero-small-left) * -1);
+  width: 100vw;
+
+  @include x-large-and-above-screen {
+    left: calc(var(--hero-large-content-left) * -1);
+    padding-left: var(--spacing-s);
+    z-index: 1;
+    width: 100vw;
+  }
+}
+
+.hero-bg-image {
+  aspect-ratio: 16 / 9;
+  background-repeat: no-repeat;
```

Comment:
During a quick test I noticed no effect from this rule. Though I only tested on small laptop window and chrome mobile emulator. Maybe it's more safe to keep it at no-repeat?